packages/amplify-e2e-core/src/init/initProjectHelper.ts

```diff
@@ -49,7 +49,13 @@ export function initJSProjectWithProfile(cwd: string, settings?: Partial<typeof
   if (s?.name?.length > 20) console.warn('Project names should not be longer than 20 characters. This may cause tests to break.');
 
   return new Promise((resolve, reject) => {
-    const chain = spawn(getCLIPath(), cliArgs, { cwd, stripColors: true, env, disableCIDetection: s.disableCIDetection })
+    const chain = spawn(getCLIPath(), cliArgs, {
+      cwd,
+      stripColors: true,
+      env,
+      disableCIDetection: s.disableCIDetection,
+      noOutputTimeout: 10 * 60 * 1000,
```

Comment:
Does `init` take 10 minutes? Searchable tests may need longer timeouts too.

Reply:
Let's keep an eye on that -- I'm not seeing failures in searchable tests, so it may be that I am barking up the wrong tree entirely.

Reply:
Searchable tests usually takes longer time during `amplify push` which was resolved previously. The init process time should stay the same as the others